content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -20,12 +20,12 @@ This guide demonstrates how to deploy an application to Deno Deploy in conjuncti
 
 ## Prerequisites
 
-- a free [GitHub](https://github.com/) account
+- a free [Prisma Data Platform](https://console.prisma.io) account
 - a free [Deno Deploy](https://deno.com/deploy) account
 - a PostgreSQL database
 - Node.js & npm installed
-- git installed
 - Deno v1.29.4 or later installed. [Learn more](https://deno.land/manual/getting_started/installation).
+- (Recommended) Latest version of Prisma ORM.
 - (Recommended) Deno extension for VS Code. [Learn more](https://deno.land/manual/getting_started/setup_your_environment#visual-studio-code).
 
 ## 1. Set up your application
```

Comment:
We could update this section to use `prisma migrate dev` over `prisma db push` in Step 2

Reply:
Yes we can